app-desktop/src/jvmMain/kotlin/social/androiddev/desktop/Main.kt

```diff
@@ -24,8 +24,8 @@ import kotlinx.coroutines.Dispatchers
 import org.koin.core.context.startKoin
 import social.androiddev.common.di.appModule
 import social.androiddev.common.theme.DodoTheme
-import social.androiddev.root.composables.RootContent
-import social.androiddev.root.navigation.DefaultRootComponent
+import social.androiddev.root.root.DefaultRootComponent
+import social.androiddev.root.root.RootContent
```

Comment:
also moved root content to separate module (for consistency with `splash` module)